File: C# I/03.Operators-And-Expressions/10.PointCircleRectangle/PointCircleRectangle.cs
using System;
    class PointCircleRectangle
    {
        static void Main()
        {
            double x = double.Parse(Console.ReadLine());
            double y = double.Parse(Console.ReadLine());
            double distance = Math.Pow(x - 1, 2) + Math.Pow(y - 1, 2);
            double circleRadius = 1.5;
            string inCircle = "outside circle";
            string inRectangle = "inside rectangle";

            if (distance <= + Math.Pow(circleRadius, 2))
            {
                inCircle = "inside circle";
            }

            if (y > 1 || y < -1 || x < -1 || x > 5)
            {
                inRectangle = "outside rectangle";
            }
            
            Console.WriteLine("{0} {1}", inCircle, inRectangle);
            
        }
    }

File: JS Fundamentals/01.Operators-And-Expressions/PointCircleRectangle.js
function solve(args) {
    let x = Number(args[0]);
    let y = Number(args[1]);
    let isInCircle;
    let isInRectangle;
    let distanceFromCenter = (Math.pow(x - 1, 2)) + (Math.pow(y - 1, 2));
    let radius = 1.5;

    if (distanceFromCenter <= Math.pow(radius, 2)) {

        isInCircle = "inside circle ";
    }

    else {
        isInCircle = "outside circle ";
    }

    if (x < -1 || x > 5 || y > 1 || y < -1) {
        isInRectangle = "outside rectangle";
    }

    else {
        isInRectangle = "inside rectangle";
    }
    console.log(isInCircle + isInRectangle);
}
solve(['2.5', '2']);